roxctl/maincommand/command.go

```diff
@@ -19,10 +19,11 @@ import (
 	connectivitymap "github.com/stackrox/rox/roxctl/connectivity-map"
 	"github.com/stackrox/rox/roxctl/declarativeconfig"
 	"github.com/stackrox/rox/roxctl/deployment"
-	"github.com/stackrox/rox/roxctl/generate"
+	generateL1 "github.com/stackrox/rox/roxctl/generate"
```

Comment:
what is `L1` in this context?

Reply:
generate netpol -> generate is level 1, netpol level 2
netpol generate -> netpol is level 1, generate level 2